indico/htdocs/js/indico/jquery/clearableinput.js

```diff
@@ -31,15 +31,16 @@
         _create: function() {
             var self = this;
 
-            self.buttonBox = $('<span class="button-box"></span>');
-            self.clearIcon = $('<a class="input-clear icon-close"></a>')
+            self.buttonBox = $('<span>').addClass('button-box');
```

Comment:
Me neither. Feel free to change it in the next round of improvements. Gotta move on.